user_templates/mcu_defaults/toolhead/overrides/BTT_SB2240_v1.0_overrides.cfg

```diff
@@ -0,0 +1,15 @@
+#-------------------------#
+#   TOOLHEAD MCU board    #
+#-------------------------#
+
+# Those overrides are automatically added when you select the SB2240 
+# toolhead MCU during the installation process. They should provide a
+# good base to work with. Feel free to adapt to your board if needed!
+
+
+# Since ADXL shares pins with the SPI interface of the TMC, it must be set up here, since different pin aliases must not resolve to the same pin
+[tmc2240 extruder]
+cs_pin: toolhead:MCU_MOTOR_SPI_NSS
+spi_software_sclk_pin: toolhead:ADXL_SCLK
+spi_software_mosi_pin: toolhead:ADXL_MOSI
+spi_software_miso_pin: toolhead:ADXL_MISO
```

Comment:
As per discussion in other PRs, as this is some pin mappings, please add them directly to the MCU file. You can also add the default overrides that you can find in this file as well: https://github.com/Frix-x/klippain/blob/develop/user_templates/mcu_defaults/toolhead/overrides/default.cfg